### ISSUE TITLE
Weird Issue: keras_contrib.layers.normalization.instancenormalization

### ISSUE THREAD

**Issue by lindalibjchn** [keras-team/keras-contrib#510]
I have already installed keras-contrib successfully, and "from keras_contrib.layers.normalization.instancenormalization import InstanceNormalization" there is no any issues for this command. but still has an issue, please check the screenshot below. this programme I ran before, it worked very well.

![image](https://user-images.githubusercontent.com/44066194/61792628-1c120500-ae15-11e9-9056-d739c7ffc4a2.png)

**Comment by lindalibjchn** [comment]
My suggestion: 

it is better to use "sudo pip install git+https://www.github.com/keras-team/keras-contrib.git"
The two ways cannot work probably. 
way1: 
git clone https://www.github.com/keras-team/keras-contrib.git
cd keras-contrib
python setup.py install
way2. 
git clone https://www.github.com/keras-team/keras-contrib.git
cd keras-contrib
python convert_to_tf_keras.py
USE_TF_KERAS=1 python setup.py install

Because the both two ways I used on google colab, the issue cannot be fixed.

**Comment by sneh-debug** [comment]
@lindalibjchn it worked on google colab?
